src/pages/iou/request/step/IOURequestStepParticipants.tsx

```diff
@@ -121,12 +133,16 @@ function IOURequestStepParticipants({
             }
 
             const firstParticipantReportID = val.at(0)?.reportID;
-            const rateID = DistanceRequestUtils.getCustomUnitRateID(firstParticipantReportID);
             const isInvoice = iouType === CONST.IOU.TYPE.INVOICE && isInvoiceRoomWithID(firstParticipantReportID);
             numberOfParticipants.current = val.length;
-
             setMoneyRequestParticipants(transactionID, val);
-            setCustomUnitRateID(transactionID, rateID);
+
+            if (!isMovingTransactionFromTrackExpense) {
+                // When moving the transaction, keep the original rate and let the user manually change it to the one they want from the workspace.
+                // Otherwise, select the default one automatically.
```

Comment:
Can we please re-word this to match the way the condition reads? I read it as "if not moving transaction from track expense" then "when moving the transaction". At that point my brain hurts because it seems to be a contradiction.

So maybe the comment can start with "If not moving the transaction from track expense, select the default rate automatically. Otherwise, keep the original P2P rate and let the user manually change it..."